backend/EkoFunkcje/EkoFunkcje/Utils/PolishCharactersTranslator.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EkoFunkcje.Utils
+{
+    public interface IStringTranslator
+    {
+        string TranslatePolishCharsInString(string dirtyString);
+    }
+
+    public class PolishCharactersTranslator : IStringTranslator
+    {
+        private Dictionary<char, char> _translations = new Dictionary<char, char>
+        {
+            {'ą', 'a'},
+            {'Ą', 'A'},
+            {'ć', 'c'},
+            {'Ć', 'C'},
+            {'ę', 'e'},
+            {'Ę', 'E'},
+            {'ł', 'l'},
+            {'Ł', 'L'},
+            {'ń', 'n'},
+            {'Ń', 'N'},
+            {'ó', 'o'},
+            {'Ó', 'O'},
+            {'ś', 's'},
+            {'Ś', 'S'},
+            {'ż', 'z'},
+            {'Ż', 'Z'},
+            {'ź', 'z'},
+            {'Ź', 'Z'}
+        };
+
+        private string TranslatePolishString(string polishLettersOnly)
```

Comment:
Z tego co widze, to ta metoda jest chyba nie uzywana i mozemy usunąć? :)

Reply:
Usunięte